index.js

```diff
@@ -13,12 +16,28 @@ if(process.env.NODE_ENV!="production"){
 }
 
 const app = express();
-
+var bodyParser = require('body-parser')
+app.use(bodyParser.urlencoded({
+  extended: true
+}));
```

Comment:
why bodyparser urlencoded and express urlencoded one with extended as false and one as true